src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="field_table">
+        <legend>Personal information:</legend>
+        <label>
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="field__info">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="field__info">
+          How old are You?
+          <input
+            type="number"
+            value="12"
+            name="old"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="field__info">
+          Full date of birth:
+          <input type="date" name="birthday">
+        </label>
+        <label class="field__info">
+          I accept the term of the agreement
+          <input type="checkbox">
```

Comment:
don't forget about `name` attribute

Reply:
Not fixed. Check the requirements one more time, please 
<img width="507" alt="image" src="https://user-images.githubusercontent.com/99403198/222890465-d0b02797-0a98-4031-a84c-ce1233a03a93.png">